sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -495,6 +495,11 @@ namespace Azure { namespace Storage { namespace Blobs {
      * @brief If the destination blob should be sealed. Only applicable for Append Blobs.
      */
     Azure::Nullable<bool> ShouldSealDestination;
+
+    /**
+     * @brief If the destination blob should copy the properties of the source
+     */
+    Azure::Nullable<bool> ShouldCopySourceBlobProperties;
```

Comment:
It looks to me this value should be either true or false. What does null mean here? What will happen the HTTP header is not included in the request?

Reply:
It's an optional header, it's default by true if not included in HTTP header

Reply:
Then I guess this should not be nullable.

```suggestion
    bool ShouldCopySourceBlobProperties = true;
```